sample-server/index.js

```diff
@@ -139,17 +136,19 @@ app.get('/fetch-score', async (req, res) => {
 
   //Get the token of the session from the headers
   let token = req.headers["x-token"];
-  if(!token) {
+  if (!token) {
     res.status(400).send({success:false, error:'Missing required header X-Token'});
     return;
   }
-  adminHeader['X-Incode-Hardware-Id'] = token;
```

Comment:
@rafaelCamaraIncode By doing this you actually modified the adminToken for the whole app, remember this is in backend, the changes you do to this value affects all other clients. I missed this in the first review

Reply:
You are totally right @levhitaIncode, sorry for the implementation, I didn't pay attention to the use of admin token in first time. Thanks for the correction.